### PR TITLE
Upload artifacts

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Notes:
- investigating #5

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
Welp, let's try it

File: conda-forge.yml [hunk 1]
```diff
@@ -4,3 +4,5 @@ github:
 conda_forge_output_validation: true
 conda_build:
   pkg_format: '2'
+azure:
+  store_build_artifacts: true
```
Comment (jakirkham) [inline conda-forge.yml:8]: Would suggest we drop this from `main` since this was primarily for debugging in the PR. Also would need a re-render

Comment (bollwyvl) [reply conda-forge.yml:8]: Yerp, was going to circle back around on it tomorrow, but this merge resulted in:

```
zstd.ZstdError: zstd compress error: Allocation error : not enough memory
```

Comment (jakirkham) [reply conda-forge.yml:8]: Sounds good

Yep known issue ( https://github.com/conda/conda-package-handling/issues/167 )

**Pull request merged**